src/main/kotlin/nomic/data/DatabaseProvider.kt

```diff
@@ -0,0 +1,18 @@
+package nomic.data
+
+import org.ktorm.database.Database
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class DatabaseProvider {
+
+    @Bean
```

Comment:
Since this @Bean seems to be breaking tests.  Lets pull it for now and pop it back in once a suitable workaround has been made